druid/src/event.rs

```diff
@@ -439,8 +467,19 @@ impl LifeCycle {
 }
 
 impl InternalLifeCycle {
-    /// Whether this event should be sent to widgets which are currently not visible
-    /// (for example the hidden tabs in a tabs widget).
+    /// Whether this event should be sent to widgets which are currently not visible and not
```

Comment:
ditto, I don't think we need to duplicate this everywhere.

Reply:
I agree